packages/react-brand-designer/src/brand-designer/brand-controls.tsx

```diff
@@ -56,9 +57,10 @@ export const BrandControls: FC<BrandControlsProps> = (opts) => {
         Brand logo
       </Title>
       <Description>PNG or JPG with a width of 140px or more</Description>
-      <ChooseFileButton onClick={() => setIsLogoModalOpen(true)}>
-        Upload logo
-      </ChooseFileButton>
+      <UploadLogoControls
+        logo={config.logo}
+        onClick={() => setIsLogoModalOpen(true)}
```

Comment:
can we not have anon functions?  this will always be a new prop and `UploadLogoControls` will always re-render

Reply:
In this instance it should be ok since `UploadLogoControls` is super tiny. A re-render here won't really have a significant impact on performance. https://reactjs.org/docs/faq-functions.html#is-it-ok-to-use-arrow-functions-in-render-methods

In this case I would prefer to leave it as an arrow function to keep the visual noise small. If you feel really strongly I can extract it into a const.